Bootstrap.php

```diff
@@ -205,6 +206,9 @@ public function subscribeEvents()
         $this->subscribeEvent('Shopware_Components_Password_Manager_AddEncoder', 'onAddPasswordEncoder');
 
         $this->subscribeEvent('Enlight_Controller_Action_PostDispatch', 'onPostDispatch', 110);
+
+        $this->subscribeEvent('Shopware_Console_Add_Command', 'onStartDispatch');
```

Comment:
Why onStartDispatch?

Reply:
You have also missed the update method

Reply:
I don't think it is missing https://github.com/shopwareLabs/SwagMigration/blob/master/Bootstrap.php#L115 although I might have tested it with a reinstall instead of an update.